_includes/markdown/Web-Vitals.md

```diff
@@ -0,0 +1,293 @@
+
+LCP is measured in seconds (s) and can be tracked against the follow DOM elements:
+
+* `<img>`
+* `<image>` - inside an SVG
+* `<video>`
+* An element with a `background-image`
+* Any element that is considered to be block-level (`display: block`)
+
+### How to diagnose Largest Contentful Paint
+The quickest way to diagnose the Largest Contentful Paint element on the page is by following these steps:
+
+1. Open _Google Chrome_
+2. Open _Chrome DevTools_
+3. Select the _Performance_ Tab
+4. Check the _Web Vitals_ checkbox
+5. Click the _Reload_ button or hit `⌘ ⇧ E` shortcut
+6. Scroll down to _Timings_
+7. Select the green _LCP_ marker
+8. In _Summary_ scroll down to "Related Node"
+9. Click the _node_ listed and it will be highlighted in the _DOM_.
+
+### How to fix Largest Contentful Paint
+
+Once you have diagnosed which element on the page has the Largest Contentful Paint, the next step is to figure out why. 
+There are 3 main factors that contribute to LCP:
+
+1. Slow server response times.
+2. Render-blocking JavaScript and CSS.
+3. Resource load times.
+
+It's important that your server is optimized in a way that doesnt have a domino effect on other vitals. 
+To measure the "speed" of your server you can track the [Time to First Byte (TTFB)](https://web.dev/time-to-first-byte) vital.
+
+Here are some high-level guidelines for ensuring Largest Contentful Paint occurs as fast as possible:
+
+* Are assets (Images, JavaScript, CSS, Video) served over a CDN?
+* Is there a sufficient and well-thought out caching strategy on the webserver?
+* Use `<link rel="preconnect">` and `<link rel="dns-prefetch">` for assets that originate at third-party domains. 
+* Ensure that scripts and styles are carefully audited to ensure that there are no render-blocking patterns in order to improve First Contentful Paint, which will consequently improve Largest Contentful Paint.
+* Ensure that your CSS bundles are minified (see [Task Runners](https://10up.github.io/Engineering-Best-Practices/tools/#task-runners)) and deferred if the CSS rules do not apply above the fold. You can also use Chromes "Coverage" tab to identify just how much of your CSS bundle is being utilized on the page.
+* Ensure that your JS bundles are minifed, compressed and if the functionality is not required above-the-fold, lazy-loaded.
+
+The time it takes the browser to fetch resources like images or videos can also have an effect on LCP:
+* Optimize and compress all images on the site - ensure images are not greater than twice their contained real-estate.
+* Make sure that images are being served over a CDN, you're serving formats like WebP or AVIF and you're using responsive images techniques.
+* For images that find themselves in Hero components, `preload` the image resource ahead of time using `<link rel="preload" href="/path/to/image" as="image" />`. For [responsive images](https://web.dev/preload-responsive-images/) you will need to add the `imagesrcset` and `imagesizes` attribute
```

Comment:
@dainelmawer 

Aren't they `srcset` and `sizes` attributes?

Reply:
@joesnellpdx per the spec these two attributes are called `imagesrcset` and `imagesizes` only when placed on a `<link>` tag :)

Reply:
Ahh... I see what you are saying...

Maybe add an example that makes that more clear?:

`  <link rel="preload"
        as="image" 
        imagesrcset="
          image-400.jpg 400w, 
          image-800.jpg 800w, 
          image-1600.jpg 1600w" 
        imagesizes="100vw">`